src/go/install.sh

```diff
@@ -270,6 +270,8 @@ if [[ "${TARGET_GO_VERSION}" != "none" ]] && [[ "$(go version 2>/dev/null)" != *
     fi
     curl -fsSL -o /tmp/go.tar.gz.asc "https://golang.org/dl/go${TARGET_GO_VERSION}.linux-${architecture}.tar.gz.asc"
     gpg --verify /tmp/go.tar.gz.asc /tmp/go.tar.gz
+    echo "Removing existing Go ${TARGET_GOROOT}..."
+    rm -rf $TARGET_GOROOT/*
     echo "Extracting Go ${TARGET_GO_VERSION}..."
```

Comment:
Can we bump the patch version? Thanks! 

https://github.com/devcontainers/features/blob/6ec70040b00d7aebcca70f97fa0611f3b4101c43/src/go/devcontainer-feature.json#L3